core/src/test/java/org/codice/imaging/nitf/core/common/DateTimeParserTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ */
+package org.codice.imaging.nitf.core.common;
+
+import java.text.ParseException;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.Mockito;
+import uk.org.lidalia.slf4jtest.TestLogger;
+import uk.org.lidalia.slf4jtest.TestLoggerFactory;
+
+/**
+ * Tests for DateTimeParser class
+ */
+public class DateTimeParserTest {
+    TestLogger logger = TestLoggerFactory.getTestLogger(DateTimeParser.class);
+
+    public DateTimeParserTest() {
+    }
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Test
+    public void testDateParsingWithoutReaderVersion() throws ParseException {
+        AbstractNitfSegmentParser parser = Mockito.mock(AbstractNitfSegmentParser.class, Mockito.CALLS_REAL_METHODS);
+        NitfReader mockReader = Mockito.mock(NitfReader.class);
+        parser.reader = mockReader;
+        Mockito.when(mockReader.getFileType()).thenReturn(FileType.UNKNOWN);
```

Comment:
In most of our other tests (except AbstractParserTest), we've been doing a static import of Mockito.when, Mockito.verify, etc.  That's the typical usage of those methods and we should be consistent with it.